core/proto/core.proto

```diff
@@ -0,0 +1,33 @@
+syntax = "proto2";
+
+package core;
+
+option go_package = "proto";
+
+message Challenge {
+	optional int64 id = 1;
+	optional string type = 2;
+	optional string status = 6;
+	optional string uri = 9;
+	optional string token = 3;
+	optional string accountKey = 4; // jose.JsonWebKey.MarshalJSON()
```

Comment:
Why not transfer this as a `bytes` since this is the native format for JSON in golang?